controllers/test

```diff
@@ -0,0 +1 @@
+make: Nothing to be done for `test'.
```

Comment:
Why is this file here?

Reply:
I'm not sure how that happened. I'll delete it